lilypad/server/models/spans.py

```diff
@@ -71,7 +71,8 @@ def _convert_span_table_to_public(
         elif span.scope == Scope.LLM:
             data = span.data
             display_name = f"{data['attributes']['gen_ai.system']} with '{data['attributes']['gen_ai.request.model']}'"
-        else:
+        else: # pragma: no cover
+            # This should never happen. Because the scope is an Enum, it should always be one of the two values. And, SQLModel should enforce this.
```

Comment:
if this is the case, why don't we just put the above `elif` block inside an `else` block instead?